install_linux.sh

```
@@ -0,0 +1,167 @@
+if [[ -d "$SIDE_DIR" ]]; then
+    ok "Side-Step directory exists: $SIDE_DIR"
+else
+    echo "  Cloning Side-Step..."
+    git clone "https://github.com/koda-dernet/Side-Step.git" "$SIDE_DIR"
+    ok "Cloned to $SIDE_DIR"
+fi
+
+# ── Install Python 3.11 via uv ─────────────────────────────────────
+step "Ensuring Python 3.11 is available"
+
+uv python install 3.11 2>/dev/null && ok "Python 3.11 ready" || warn "Assuming Python 3.11 is already available"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)